Controllers/SubscriptionsController.cs

```diff
@@ -101,5 +101,26 @@ public async Task<Subscription> RemoveSubscription(string userId, [FromBody] Sub
 
             return subscription;
         }
+
+        /// <summary>
+        /// Remove a subscription
+        /// </summary>
+        /// <param name="userId"></param>
+        /// <param name="subscription"></param>
+        /// <returns></returns>
+        [HttpDelete]
+        [Route("{userId}/sub/removeAll")]
+        public async void RemoveSubscriptions(string userId)
+        {
+            var user = await GetOrCreate(userId);
+            user.Subscriptions.ForEach(delegate (Subscription sub)
+            {
+                subEngine.Unsubscribe(sub);
+            });
```

Comment:
```suggestion
            user.Subscriptions.ForEach( sub=>
            {
                subEngine.Unsubscribe(sub);
            });
```
you could use the arrow function